README.md

```diff
@@ -1 +1,40 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello-world
+
```

Comment:
- [ ] I think it would be better to change the project title from **Hello World** to **Hello Microverse** as it is the project name you built. And you can do the same with the **repository's name** if you don't know how to rename a repo click [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) to get hints.